hotshot-state-prover/src/mock_ledger.rs

```diff
@@ -478,8 +478,8 @@ pub fn gen_plonk_proof_for_test(
         .iter()
         .zip(prove_keys.iter())
         .enumerate()
-        .for_each(|(i, (cs, pk))| {
-            let extra_msg = Some(format!("extra message: {}", i).into_bytes());
+        .for_each(|(_, (cs, pk))| {
+            let extra_msg = Some(String::new().into_bytes()); // We set extra_msg="" for the contract tests to pass
```

Comment:
simply `Some(vec![])` is enough, no need for string->bytes conversion.

Reply:
Right, done in 43171a81.